app/src/functions/notify-endpoint-usage/app.ts

```diff
@@ -87,22 +125,70 @@ async function getUserThresholdExceeded(appData: ApplicationData[]) {
   return extendedAppData
 }
 
+async function getLoadBalancerThreshold(appData: ApplicationData[], dbApps: IApplication[], loadBalancers: ILoadBalancer[]): Promise<ExtendedLoadBalancerData> {
+  const extendedLBData: ExtendedLoadBalancerData = {}
+
+  appData.forEach(async app => {
+    const dbApp = dbApps.find((data => data.freeTierApplicationAccount.address === app.address))
+
+    if (dbApp === undefined) {
+      return
```

Comment:
i know its wip but we probably want a log here or something... Not exactly sure what, but definitely some alert